src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, value) {
+      callback(value);
+
+      return this;
+    },
+
+    add(value) {
+      calculator.result += value;
+    },
```

Comment:
try to rewrite this way. Because if you change variable name, you should add a lot of changes.
With this solution better in 40 line return object without creating variable.
```suggestion
    add(value) {
      this.result += value;
    },
```